django/thunderstore/core/settings.py

```diff
@@ -77,6 +77,7 @@
     CELERY_EAGER_PROPAGATES_EXCEPTIONS=(bool, False),
     REPOSITORY_MAX_PACKAGE_SIZE_MB=(int, 500),
     REPOSITORY_MAX_PACKAGE_TOTAL_SIZE_GB=(int, 1000),
+    SESSION_COOKIE_DOMAIN=(str, ""),
```

Comment:
The second param is the default, in this case we probably want it to be `None` to fall back to Django defaults

Reply:
When you retrieve the value with `env.str("SESSION_COOKIE_DOMAIN")` it gets turned into an string. Also if you use `env.bool("SESSION_COOKIE_DOMAIN")` it returns False even when it contains something or is None :---) I could write a resolver for these kinds of situations, but seems like an overkill to do that.

Reply:
Gotcha, so let's go with the solution proposed in https://github.com/thunderstore-io/Thunderstore/pull/360#discussion_r700968931

Reply:
Answered to this on the previous comment.